Formula/scalariform.rb

```diff
@@ -1,11 +1,11 @@
 class Scalariform < Formula
   desc "Scala source code formatter"
-  homepage "https://github.com/daniel-trinh/scalariform"
-  url "https://github.com/daniel-trinh/scalariform/releases/download/0.1.7/scalariform.jar"
-  sha256 "8cb179402a7960310a8c6639a20ab94277ec3052cb75ea3ddba31265d070f0d6"
+  homepage "https://github.com/scala-ide/scalariform"
```

Comment:
The project switched its home, see https://github.com/daniel-trinh/scalariform#notice-weve-moved